ConductionNL/docparser
# development

- Set default port to 80
- Add parsing by URL
- Subpath ingress
- Take version into account on deploy
- Up
- Added postman collection
- build
- updated postman collection
- Add postman checking
- Uups
- Add OpenZaak componenten
- Auto deploy to dev zuid-drecht
- Bump symfony/http-kernel from 4.4.8 to 4.4.13 in /api
- NginX Fix
- Fixjes en cgb update
- Let's see if this works
- Update postman
- Update CGB
- Helm fix
- Helm
- Update CGB
- Another CGB update
- CGB update
- Test workflow change
- Let's try it like this
- also push
- fixje?
- fixje!
- another fix attempt
- Another try on tagging without building
- Remove some merge conflict
- remove unnecessary )
- Add condition to tagging
- Check if tags are set correctly
- Is APP_BUILD correct?
- Export fixje
- more fixes
- foutje
- Try with different location
- only tag conduction images
- Turn around tags
- re-enable pushing
- Also push versioned images
- Uppercase
- small fix on ingress
- Meertaligheid van resources
- Fix Gedmo
- revert gedmo fix, upgrade symfony/flex
- Updates
- Clean up
- Add helm repo
- Fixes
- Apply fixes from StyleCI
- auto doc update
